app/javascript/packs/stylesheets/bootstrap.scss

```diff
@@ -200,6 +200,7 @@ select.form-control {
   // will be drawn in front of any other modals
   background-color: rgba(20, 20, 20, 0.60);
   z-index: 2000;
+  white-space: pre-line;
```

Comment:
Why this change?

Reply:
I added this so I could put `\n` into the message